lib/resty/redis.lua

```diff
@@ -67,6 +67,14 @@ function _M.new(self)
 end
 
 
+function _M.register_module_prefix(mod)
+    _M[mod] = function(self)
+        self._module_prefix = mod .. "."
```

Comment:
```suggestion
        self._module_prefix = mod
```
move the `.. "."` to line 414: https://github.com/openresty/lua-resty-redis/pull/211/files#diff-adb515948c05651051c83fd6fb3ce7ceR414
to avoid creating a new string for better performance.

Reply:
@doujiang24 
There will be more new strings, as each time the `cmd` argument need to be a concatenated string in line 414.

Reply:
@spacewander  I don't think so.
it will generate a new string when calling the module function, like `redis:bp()`.
and generate another string when calling the method, like `:add()`.

that means will generate two string in `redis:bp:add()`.

while `module_prefix .. "." .. cmd` will generate one string.